knotx-example/knotx-example-app/src/test/resources/test.io.knotx.KnotxServer.json

```diff
@@ -25,7 +25,7 @@
       "routing": {
         "GET": [
           {
-            "path": "/content/.*",
+            "path": ".*\\.html",
```

Comment:
Good eye @toniedzwiedz , however in this case it works differently.
This path is used in [KnotxServerVertice](https://github.com/Cognifide/knotx/blob/16c6479cc8f47d83c1628dc0414df4c31460e676/knotx-server/src/main/java/io/knotx/server/KnotxServerVerticle.java#L58) to provide proper routing. When you check [Route.pathRegex](http://vertx.io/docs/apidocs/io/vertx/ext/web/Route.html#pathRegex-java.lang.String-) method documentation it says:
```
Set the path prefix as a regular expression. If set then this route will only match request URI paths, the beginning of which match the regex. Only a single path or path regex can be set for a route.
```

So the part important here is only a request path, not whole request. And this works fine with this prefix :)